claudia-lab/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "claudia-lab",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts" : {
+    "test": "DEBUG='fruit*' ./node_modules/mocha/bin/mocha",
+    "start": "DEBUG='fruit*' node server.js"
+  },
+  "dependencies": {
+    "bluebird": "^3.4.6",
+    "body-parser": "^1.15.2",
+    "debug": "^2.2.0",
+    "express": "^4.14.0",
+    "http-errors": "^1.5.0",
+    "mocha": "^3.0.2",
+    "morgan": "^1.7.0",
+    "node-uuid": "^1.4.7"
+  },
+  "devDependencies": {
+    "eslint": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "chai": "^3.5.0",
+    "mocha": "^3.1.0",
+    "superagent": "^2.3.0"
```

Comment:
@loomnugget nice job on the separation of dev dependencies and actual application dependencies - one of the first things I look for in this file, when reviewing code in interviews, is the proper delineation between the modules that belong in these sections - your's looks good.